ParabolInc/parabol
# feat(orgAdmin): Add org admin teaser in org team page for non-enterprise orgs

# Description

Fixes #10158 

## Demo

<img width="1179" alt="Screenshot 2024-09-26 at 14 24 57" src="https://github.com/user-attachments/assets/9a1183a2-54ce-42f6-9244-5d03c266c3d7">

## Testing scenarios

- [ ] Enterprise org
  - Go to `http://localhost:3000/me/organizations/xxxx/teams` for an `enterprise` org
  - See no teaser

- [ ] Starter og
  - Go to `http://localhost:3000/me/organizations/xxxx/teams` for a `starter` org
  - See the teaser

## Final checklist
